api1/api1.js

```diff
@@ -42,20 +42,17 @@ app.post("/api1", (req, res) => {
   let insuranceCost = 0;
   let model = req.body.model;
   let year = req.body.year;
-  // let carModel = model.toLowerCase().split('');
-  // insert function here
-  console.log(`model: ` + model);
+  // console.log(`model: ` + model);
   if (!regExYear.test(String(year))) {
     console.log("error");
```

Comment:
I see other console.log() lines deleted by this commit; did intend to remove this too?